src/npr/communication.py

```diff
@@ -222,8 +222,9 @@ def query_parkour(config, flowcell, msg):
             print(parkour_dict)
             first_key = list(parkour_dict.keys())[0]
             first_entry = list(parkour_dict[first_key].keys())[0]
-            org = parkour_dict[first_key][first_entry][-3]
-            organism = get_organism_name(org)
+            org = parkour_dict[first_key][first_entry][3][0]
```

Comment:
so `org` is something like "H. sapiens (...)", right?

Reply:
ah, wait... you also went from -3 to +3, I'm super confused now :P Let me come back to this tomorrow, we can have a look together.